CHANGES.rst

```diff
@@ -18,10 +20,10 @@ dev (master)
 
 * ``pip install urllib3[secure]`` will install Certifi and
   PyOpenSSL as dependencies. (Issue #678)
-  
+
```

Comment:
I love how you re-spaced all the old changes, but you didn't add a space to your new entry. :P